apps/web-giddh/src/app/shared/header/components/account-update-new-details/account-update-new-details.component.html

```diff
@@ -374,7 +374,7 @@ <h3>Address {{i+1}}</h3>
                 <button type="button" *ngIf="showDeleteButton" class="btn btn-danger " (click)="deleteClicked.emit(true)">
                     Delete
                 </button>
-                <button (click)="submit()" class="btn btn-primary mrL1 updateBtnZindex" [ladda]="createAccountInProcess$ | async" [disabled]="addAccountForm.invalid">Update
+                <button (click)="submit()" class="btn btn-primary mrL1 updateBtnZindex" [ladda]="createAccountInProcess$ | async" [disabled]="addAccountForm.invalid || !(isGstValid$ | async)">Update
```

Comment:
line 156 refers to `isGstValid` either revert the observable change and keep using variable or handle this code on line 156